<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAudioSetupComponent</class>
 <widget class="QFrame" name="CAudioSetupComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio setup</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AudioSetup">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QFormLayout" name="fl_SetupAudio">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_Info">
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_ExtraInfo">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>extra info goes here</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_SetupAudioInputDevice">
       <property name="text">
        <string>In</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_SetupAudioInputDevice">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_SetupAudioOutputDevice">
       <property name="text">
        <string>Out</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_SetupAudioOutputDevice">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_SetupAudioLoopback">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioLoopback">
       <property name="text">
        <string>Loopback, test sound in- to output</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_SoundDirectory">
       <property name="text">
        <string>Dir.:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QWidget" name="wi_SoundDir" native="true">
       <layout class="QHBoxLayout" name="hl_SoundDir" stretch="0,0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="le_SoundDir">
          <property name="placeholderText">
           <string>your private sound directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SoundReset">
          <property name="text">
           <string>reset</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pb_SoundDir">
          <property name="text">
           <string>[...]</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_Notifications">
       <property name="toolTip">
        <string>Notifications</string>
       </property>
       <property name="text">
        <string>Not.:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessagePrivate">
       <property name="text">
        <string>notification for private text messages</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessageSupervisor">
       <property name="text">
        <string>supervisor messages</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioNotificationTextCallsignMentioned">
       <property name="text">
        <string>notfication for text msg. with my callsign</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioNotificationVoiceRoomJoined">
       <property name="text">
        <string>notification joining a voice room</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioNotificationVoiceRoomLeft">
       <property name="text">
        <string>notification leaving a voice room</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioNoTransmission">
       <property name="text">
        <string>No audio transmission warning</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioPTTClickDown">
       <property name="text">
        <string>PTT click (key down)</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioPTTClickUp">
       <property name="text">
        <string>PTT click (key up)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vs_AudioSetup">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>143</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_ExtraInfo</tabstop>
  <tabstop>cb_SetupAudioInputDevice</tabstop>
  <tabstop>cb_SetupAudioOutputDevice</tabstop>
  <tabstop>cb_SetupAudioLoopback</tabstop>
  <tabstop>le_SoundDir</tabstop>
  <tabstop>pb_SoundReset</tabstop>
  <tabstop>pb_SoundDir</tabstop>
  <tabstop>cb_SetupAudioNotificationTextMessagePrivate</tabstop>
  <tabstop>cb_SetupAudioNotificationTextMessageSupervisor</tabstop>
  <tabstop>cb_SetupAudioNotificationTextCallsignMentioned</tabstop>
  <tabstop>cb_SetupAudioNotificationVoiceRoomJoined</tabstop>
  <tabstop>cb_SetupAudioNotificationVoiceRoomLeft</tabstop>
  <tabstop>cb_SetupAudioNoTransmission</tabstop>
  <tabstop>cb_SetupAudioPTTClickDown</tabstop>
  <tabstop>cb_SetupAudioPTTClickUp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
